src/rf/apps/workers/process.py

```diff
@@ -279,26 +294,66 @@ def copy_image(self, record):
         else:
             return False
 
-    def save_result(self, record):
+    def chunk(self, record):
```

Comment:
With the exception of the status enum, this function and the mosaic function are exactly the same. Consider DRYing this up.